### PR TITLE
Masha

### PR DESCRIPTION
**Pull request by maxaalexeeva**
Some rules for units, some action adjustments, some sentences added as todos to the test files, changed placement of 'model' in the taxonomy. Details in commit messages.

### CODE REVIEW

File: text_reading/src/main/resources/org/clulab/aske_automates/grammars/entities/grammar/avoid.yml [hunk 1]
```diff
@@ -79,20 +79,20 @@ rules:
...
       [word="i.e"]
 
-  - name: "person" #doesn't work bc Kcbmin is found as PERSON
+  - name: "proper_noun" #eliminating a stand-alone 'person' doesn't work bc Kcbmin is found as PERSON
     label: Avoid
     priority: 2
     type: token
     pattern: |
-      [entity="PERSON"] [entity="PERSON"] [entity="PERSON"]
+      [entity=/PERSON|ORGANIZATION|LOCATION/] [entity=/PERSON|ORGANIZATION|LOCATION/] [entity=/PERSON|ORGANIZATION|LOCATION/]
```
Comment (BeckySharp) [inline text_reading/src/main/resources/org/clulab/aske_automates/grammars/entities/grammar/avoid.yml]: does it work to have `[entity=/PERSON|ORGANIZATION|LOCATION/]{2,3}` here?

File: text_reading/src/main/resources/org/clulab/aske_automates/grammars/units.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+  - name: var_unit_rel
+    label: Unit
+    priority: ${priority}
+    type: token
+    example: "T = daily mean air temperature [°C]"
+    action: looksLikeAUnit
+    pattern: |
+      @variable:Variable [word=/.*/]+ [word=/-LRB-|-LSB-/] (?<unit> [word = /.*/]+) (?=[word=/-RRB-|-RSB-/])
+      #false positive: "In the second approach, Kc is split into two factors that separately describe the evaporation (Ke) and transpiration (Kcb) components."
```
Comment (BeckySharp) [inline text_reading/src/main/resources/org/clulab/aske_automates/grammars/units.yml]: here's an example where a gazetteer would help, like you said

File: text_reading/src/main/scala/org/clulab/aske/automates/OdinActions.scala [hunk 3]
```diff
@@ -95,7 +95,7 @@ class OdinActions(val taxonomy: Taxonomy, expansionHandler: Option[ExpansionHand
       // The longest mention (i.e., the definition) should be at least 3 characters, else it's likely a false positive
       // todo: tune
       // todo: should we constrain on the length of the variable name??
-      if (sorted.last.text.length < 3) {
+      if (sorted.last.text.length < 3 || looksLikeAVariable(Seq(sorted.head), state).isEmpty) {
```
Comment (BeckySharp) [inline text_reading/src/main/scala/org/clulab/aske/automates/OdinActions.scala]: can you add a comment for this addition so future us remember what we did? thanks!

File: text_reading/src/main/resources/org/clulab/aske_automates/grammars/units.yml (hunk 2 above)
Comment (maxaalexeeva) [reply text_reading/src/main/resources/org/clulab/aske_automates/grammars/units.yml]: Exactly!

**Comment by maxaalexeeva** [comment]
@BeckySharp, done. Thanks for the suggestions!

**Pull request merged**